doc/sphinx/changes.rst

```diff
@@ -8,6 +8,992 @@ Recent changes
 
    .. include:: ../unreleased.rst
 
+Version 8.20
+------------
+
+Summary of changes
+~~~~~~~~~~~~~~~~~~
+
+Coq version 8.20 TODO_main_new_features
+
+We highlight some of the most impactful changes here:
+
+  - :ref:`rewrite_rules`
+
+  - :ref:`primitive strings <pstring_library>`
+
+  - A lot of work went into reducing the size of the bytecode segment,
+    which in turn means that .vo files might now be considerably
+    smaller.
+
+  - TODO other notable changes
+
+Notable breaking changes:
+
+  - TODO I can identify
+
+  - syntactic global references passed through the `using` clauses of
```

Comment:
Not so sure about that one, there is a link to about the same thing three words later and I feel `using` is more readable than `auto_using`, because it's the user visible syntax, rather than some internal parser name.

Reply:
`auto_using` appears in the doc in the syntax of (I think) all the relevant commands (listed below), so it has a nice precision to it.  Maybe say ``` `using` clauses (:n:`@auto_using`) ```?

```
| "trivial" OPT auto_using OPT hintbases
| "info_trivial" OPT auto_using OPT hintbases
| "debug" "trivial" OPT auto_using OPT hintbases
| "auto" OPT nat_or_var OPT auto_using OPT hintbases
| "info_auto" OPT nat_or_var OPT auto_using OPT hintbases
| "debug" "auto" OPT nat_or_var OPT auto_using OPT hintbases
| "eauto" OPT nat_or_var OPT auto_using OPT hintbases
| "debug" "eauto" OPT nat_or_var OPT auto_using OPT hintbases
| "info_eauto" OPT nat_or_var OPT auto_using OPT hintbases
| "dfs" "eauto" OPT nat_or_var OPT auto_using OPT hintbases
```

Reply:
Still feels rather redundant to me (since there is already a link to auto, auto_using is just one more click away anyway) but any stronger rational / opinion from anyone else is welcome.